internal/inspect/utils.go

```diff
@@ -0,0 +1,68 @@
+package inspect
+
+import (
+	"embed"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/go-errors/errors"
+	"github.com/jackc/pgconn"
+	"github.com/jackc/pgx/v4"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/supabase/cli/internal/db/reset"
+	"github.com/supabase/cli/internal/utils"
+)
+
+const ROOT_DIR = "queries/"
+const CSV_QUERY = `COPY (%s) TO STDOUT WITH CSV HEADER`
+
+//go:embed queries/*.sql
+var queries embed.FS
+
+func ReadQuery(query string) string {
+	path := fmt.Sprintf("%s%s.sql", ROOT_DIR, query)
+	queryString, err := queries.ReadFile(path)
+	if err != nil {
+		println(err.Error())
+		return ""
+	}
+	return string(queryString)
+}
+
+func Report(cmd *cobra.Command, config pgconn.Config, fsys afero.Fs, options ...func(*pgx.ConnConfig)) error {
+	queries, err := queries.ReadDir("queries")
+	if err != nil {
+		return err
+	}
+	date := time.Now().Format("2006-01-02")
+	out, err := cmd.Flags().GetString("out-path")
+	if err != nil {
+		return err
+	}
+	if out != "" {
+		err := os.MkdirAll(out, os.ModePerm)
+		if err != nil {
+			return errors.Errorf("failed to create output directory: %w", err)
+		}
+	} else {
+		out = "./"
+	}
+	fmt.Println("Running queries...")
```

Comment:
```suggestion
	fmt.Fprintln(os.Stderr, "Running queries...")
```

Progress logs should be printed to stderr by POSIX standard. https://pubs.opengroup.org/onlinepubs/9699919799/functions/stderr.html


> standard error (for writing diagnostic output)


This allows other programs to consume from piped stdout.